examples/otlp/docker/otel-collector-config-demo.yaml

```diff
@@ -0,0 +1,54 @@
+receivers:
+  opencensus:
+  otlp:
+    protocols:
+        grpc:
+  otlp/withendpoint:
```

Comment:
Unused I think, if it's just for documentation can you add a comment describing it?

Reply:
I think I can remove this. will do

Reply:
I'm actually not sure what this is for, even.

Reply:
removed